src/resources/elements/primeDesignSystem/pcard/pcard.scss

```diff
@@ -0,0 +1,20 @@
+@import "../styles";
+
+.pcard {
+  padding: $spacing-normal; // only if there is no padding on the parent
+  border-radius: $border-radius-small;
+  cursor: pointer;
```

Comment:
I'm wondering about the default state of `cursor: pointer`.
My initial reaction when checking out the demo, was "Why does this have a pointer"?

What are your thoughts on this @giladt ?

Reply:
good catch. It's a leftover from the prev version. But I agree it shouldn't be defined by the component, but rather by the actual use of it.